ui/raidboss/popup-text.js

```diff
@@ -31,6 +31,19 @@ class PopupText {
     this.alertText = document.getElementById('popup-text-alert');
     this.alarmText = document.getElementById('popup-text-alarm');
 
+    if (Options.PlayerNameOverride !== null &&
```

Comment:
This is a bit of a silly request, but can you add another option for "use act tts that defaults to true"? It's very likely that I will use this "overriding the player" logic for the raid emulator (which pretends to be a different player).  So, I'd like to remove the assumption that "using a player override" is the same case as "not using act tts".

Reply:
The only potential issue here is that if multiple people are using the same websocket for cactbot callouts, and any of them don't include the option to not `use act tts`, that player's alerts are going to route back through ACT to the person hosting causing them to get incorrect (dps vs healer, etc) or otherwise duplicate callouts with slight delay, causing a sort of echo effect.

Reply:
Ah, I see the fear here.  I would be fine with an override *parameter* also setting the act tts option, and not requiring a second parameter.  My request was more around the code inside the constructor, etc.  Yeah, it's just the constructor so in practice it doesn't matter, but I just want to decouple the two options in case it proliferates more in the future.

Reply:
So add the option, default it to true, and if the player override is specified default it to false, and finally make sure the check for the parameter happens after the check for player override to allow that parameter to still take priority?

Reply:
Yes, up to the "and finally" part and no after that.  I was thinking no parameter for act tts, just an option.  I just want to decouple the two behaviors of tts and player override in popuptext.

Reply:
Alright, I understand the intention now, but wouldn't that config just use the ACT host's user config, therefore causing the same problem as before (ACT host gets either all the callouts or none of the callouts)?

Reply:
If you set the parameter inside that same function where you're looking at parameters in raidboss.js, then that will override everything any user config happened to set.

Reply:
Please review the latest commits to make sure we're on the same page here for this functionality. It made more sense to me for it to be an opt-in-to-browser-tts option, rather than an opt-out-of-act-tts option.